src/utils/watching.ts

```diff
@@ -0,0 +1,11 @@
+import { Colony, User } from '~types';
+
+/* Find in users watchlist if watching a specific colony */
+export const watchingColony = (
+  colony: Colony | undefined,
+  user: User | null | undefined,
+): any => {
```

Comment:
You can get the return type for example by accessing `NonNullable<User['watchlist']>['items'][number]`. Alternatively create a GraphQL fragment for the fields you're querying on User > watchlist and use the fragment here, like so (`fragments/user.graphql`)
```gql
fragment WatchlistItem on WatchedColonies {
  id
  colony {
    ..WatchedColony
  }
  createdAt
}

# Later in the user fragment:
watchlist {
  items {
    ..WatchlistItem
  }
}
```

This will generate a `WatchlistItemFragment` which, in line with other types, you can export from `types/graphql` and use here